src/components/Dashboard/StaffNotation.vue

```diff
@@ -71,6 +87,19 @@
                 <span class="app-blue">Record Conversion</span>
               </v-list-item-title>
             </v-list-item>
+            <template v-if="isFirm">
+              <v-list-item v-if="isHistorical" @click="showPutBackOnDialog()" :disabled="!isHistorical">
+                <v-list-item-title>
+                  <span class="app-blue">Put Back On</span>
+                </v-list-item-title>
+              </v-list-item>
+              <v-list-item v-if="!isHistorical"
+              @click="showAdministrativeDissolutionDialog()" :disabled="disabled">
```

Comment:
Move this to previous line (or if necessary, indent it as per existing code style).